### PR TITLE
chore(deps): Replace go-ping by pro-bing

### PR DESCRIPTION
**Pull request by Hipska**
# Required for all PRs

- [x] Updated associated README.md.
- [x] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

The go-ping/ping package is going to be abandoned.

### CODE REVIEW

**Review by sspaink — APPROVED** [review]
Thanks for noticing this happened and migrating it to the new repository. Kind of a strange name lol

**Comment by SuperQ** [comment]
Yea, we're planning to expand it to support other probe types. Like TCP or HTTP. We decided to go with a silly name. :grin:

File: go.mod [hunk 1]
```diff
@@ -130,6 +129,7 @@ require (
 	github.com/pborman/ansi v1.0.0
 	github.com/pion/dtls/v2 v2.1.5
 	github.com/pkg/errors v0.9.1
+	github.com/prometheus-community/pro-bing v0.0.0-20220620120154-c0162ab18714
```
Comment (SuperQ) [inline go.mod:132]: I just tagged v0.1.0 for you. :)

Comment (Hipska) [reply go.mod:132]: Oh great, will update..

File: plugins/inputs/ping/ping.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-ping/ping"
+	probing "github.com/prometheus-community/pro-bing"
```
Comment (reimda) [inline plugins/inputs/ping/ping.go:17]: Let's keep the import name ping. That will cut down on unnecessary changes in the PR

```suggestion
	ping "github.com/prometheus-community/pro-bing"
```

File: plugins/inputs/ping/ping_test.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/go-ping/ping"
+	probing "github.com/prometheus-community/pro-bing"
```
Comment (reimda) [inline plugins/inputs/ping/ping_test.go:14]: Same here

**Review by reimda — CHANGES_REQUESTED** [review]

**Comment by reimda** [comment]
There is no notice on go-ping/ping that says it's abandoned. I see that prometheus-community/pro-bing has recent activity and I appreciate the work people are doing there. Can you share some background on why we should switch to the fork?

**Comment by Hipska** [comment]
In  their Slack Channel @SuperQ opted to archive it or at least mention it on the readme, but none of the maintainers responded to that. It is so abandoned that they even not care to inform the users 😉

**Comment by SuperQ** [comment]
Yea, that's how difficult it is. Due to the dual approval necessary, I can't open a PR to even say that it's abandoned.

If either of you could open a PR to update the README, I could try and get it merged eventually.

**Comment by reimda** [comment]
And thanks @SuperQ for keeping the ping project alive with pro-bing

**Comment by Hipska** [comment]
@SuperQ do you have something in mind?
> If either of you could open a PR to update the README, I could try and get it merged eventually.

**Pull request merged**